src/sentry/db/models/fields/jsonfield.py

```diff
@@ -142,6 +148,10 @@ def value_to_string(self, obj):
         return self.value_from_object(obj)
 
 
+class JSONFieldNoCreator(JSONField):
+    no_creator_hook = True
```

Comment:
Should we still have this if you're not using it?

Reply:
Good call, removing.